### ISSUE TITLE
[FEATURE] ADDING A SIMPLE CHAT BOT

### ISSUE THREAD

**Issue by kharshita590** [narayan954/dummygram#418]
### Description

I WANT TO ADD A SIMPLE CHAT BOT WHICH WILL PROVIDE YOU USER A GREAT EXPERIENCE

### Screenshots

..

### Additional information

,,,

**Comment by narayan954** [comment]
Thanks for raising the issue Harshita, What'd we need for that? Actually I've never integrated a chatbot so I wanted to know. Would that be done on the frontend only? Also, what kind of conversation it can give?

**Comment by JAYESHBATRA** [comment]
@narayan954 Please assign the issue to me.

**Comment by narayan954** [comment]
> @narayan954 Please assign the issue to me.

this issue is already assigned

**Comment by kharshita590** [comment]
@narayan954 I will use react js and for making logic i will use NLP libraries.

**Comment by narayan954** [comment]
> @narayan954 I will use react js and for making logic i will use NLP libraries, what kind of conversation you want that will depend upon, like if a user enters your site they can ask some questions and can take help

great, you can go ahead!

**Comment by kharshita590** [comment]
but can you give me some information how does your website works.. @narayan954

**Comment by narayan954** [comment]
> but can you give me some information how does your website works.. @narayan954

Yes sure, most of it should be available on the readme... This website has the front end UI in React, and all of the backend operations like authentication, storage are handled by firebase. We have configured this website to store the actual post media in firebase storage and enter this reference information in the firebase firestore database which can be queried quickly first to structure the website, then the media content is loaded from firebase storage. Also authentication is done through three option, all implemented for firebase, first email login, second google oauth, third facebook oauth. We can like, share and save/favourite any post... 

I think this could answer your question or what other information do you require?

**Comment by kharshita590** [comment]
@narayan954 okk

**Comment by Palakkgoyal** [comment]
Hey @narayan954 Can I work on this?

**Comment by narayan954** [comment]
> Hey @narayan954 Can I work on this?

sure, you can go ahead!
